pcd8544/pcd8544.go

```diff
@@ -0,0 +1,119 @@
+// Package pcd8544 implements a driver for the PCD8544 led matrix controller, used in Nokia's 5110 and 3310 phones.
+//
+// Datasheet: http://eia.udg.edu/~forest/PCD8544_1.pdf
+package pcd8544
+
+import (
+	"time"
+
+	"github.com/aykevl/tinygo/src/machine"
+)
+
+// Device wraps an SPI connection.
+type Device struct {
+	bus    machine.SPI
+	dcPin  machine.GPIO
+	rstPin machine.GPIO
+	scePin machine.GPIO
+	buffer [504]byte
+}
+
+// New creates a new PCD8544 connection. The SPI bus must already be configured.
+func New(bus machine.SPI, dcPin machine.GPIO, rstPin machine.GPIO, scePin machine.GPIO) *Device {
+	return &Device{
+		bus:    bus,
+		dcPin:  dcPin,
+		rstPin: rstPin,
+		scePin: scePin,
+		buffer: [504]byte{},
+	}
+}
+
+// Init initializes the LCD with default configuration
+func (d *Device) Init() {
```

Comment:
Can you rename this function to `Configure()`? That's the same name that is used by the other drivers, for consistency.